news/2023/2023-02-12-community-choice-2022-results.md

```diff
@@ -0,0 +1,189 @@
+- [**[Kyoro] - Mochio-**](https://osu.ppy.sh/beatmapsets/1473006) by [**Snow Note**](https://osu.ppy.sh/users/643394) with **33** votes.
+- [**Tarako - Bing Chilling-ish**](https://osu.ppy.sh/beatmapsets/1820912) by [**Aurorance**](https://osu.ppy.sh/users/13142180) with **32** votes.
+- [**Hoshimachi Suisei - Stellar Stellar**](https://osu.ppy.sh/beatmapsets/1680549) by [**ImWater**](https://osu.ppy.sh/users/10616849) with **30** votes.
+- [**Feryquitous vs. xi - Orca**](https://osu.ppy.sh/beatmapsets/1166748) by [**lcfc**](https://osu.ppy.sh/users/7322726) with **29** votes.
+
+## osu!taiko
+
+### Community Choice
+
+[![](/wiki/shared/news/2023-02-12-community-choice-2022-results/truth.jpg)](https://osu.ppy.sh/beatmapsets/1859338)
+
+**GTS Sound Team - \<\<\/nttld.:beings\>\> \~Truth in Uncertainty\~** is a collaboration of absolutely titanic proportions, demonstrated not only by its *enormous* contributor list (which we will proceed to share in a moment), but also by its substantial **96** vote total, leading over the closest competitor by a considerable margin.
+
+Okay, deep breaths. Here we go.
+
+A joint collaboration between the composters of the GTS Sound Team (namely [SDMNE](https://soundcloud.com/sdmne), [Sparxe](https://soundcloud.com/sparxenic), [blobdash](https://soundcloud.com/blobdash), [Raphlesia](https://soundcloud.com/raphlesia), [ptar124](https://soundcloud.com/part-lerts), [kyou1110](https://soundcloud.com/kyou1110), [II-L](https://soundcloud.com/ii-l), [rN](https://soundcloud.com/rurinn), [takehirotei](https://soundcloud.com/takehirotei) and [katagiri](https://soundcloud.com/katagiri) and the GTS Mapping Team (namely [Cychloryn](https://osu.ppy.sh/users/6921736), [Nwolf](https://osu.ppy.sh/users/1910766), [Mew](https://osu.ppy.sh/users/2345156), [gaston_2199](https://osu.ppy.sh/users/5938161), [Raphalge](https://osu.ppy.sh/users/3918650), [Ak1o](https://osu.ppy.sh/users/1600041), [Hivie](https://osu.ppy.sh/users/14102976), [Alchyr](https://osu.ppy.sh/users/4993032), [rubies87](https://osu.ppy.sh/users/4949934), [Briesmas](https://osu.ppy.sh/users/2865172), [UnagiDon](https://osu.ppy.sh/users/2062410), [Gamelan4](https://osu.ppy.sh/users/9856910), [Heaxys](https://osu.ppy.sh/users/5671417), [Ideal](https://osu.ppy.sh/users/3869519), [HiroK](https://osu.ppy.sh/users/4050738), [5\_5](https://osu.ppy.sh/users/6853438), [BabySnakes](https://osu.ppy.sh/users/4669728), [MTNTWarz](https://osu.ppy.sh/users/14036825), [Rinze](https://osu.ppy.sh/users/19901680), [cdh](https://osu.ppy.sh/users/2403621), [Cynplytholowazy](https://osu.ppy.sh/users/3901754) and [Horiiizon](https://osu.ppy.sh/users/8071438)), *\<\<\/nttld.:beings\>\> \~Truth in Uncertainty\~* is a visually stunning mapping masterpiece, helped along the way into Ranked by beatmap nominators [Axer](https://osu.ppy.sh/users/7299864) and [op45667](https://osu.ppy.sh/users/21973491).
```

Comment:
composters -> composers

Also if all composers, mappers and BNs are noted, maybe noting down designers for the background could be worthwhile? Just a suggestion here.

Reply:
```suggestion
A joint collaboration between the composers of the GTS Sound Team (namely [SDMNE](https://soundcloud.com/sdmne), [Sparxe](https://soundcloud.com/sparxenic), [blobdash](https://soundcloud.com/blobdash), [Raphlesia](https://soundcloud.com/raphlesia), [ptar124](https://soundcloud.com/part-lerts), [kyou1110](https://soundcloud.com/kyou1110), [II-L](https://soundcloud.com/ii-l), [rN](https://soundcloud.com/rurinn), [takehirotei](https://soundcloud.com/takehirotei) and [katagiri](https://soundcloud.com/katagiri) and the GTS Mapping Team (namely [Cychloryn](https://osu.ppy.sh/users/6921736), [Nwolf](https://osu.ppy.sh/users/1910766), [Mew](https://osu.ppy.sh/users/2345156), [gaston_2199](https://osu.ppy.sh/users/5938161), [Raphalge](https://osu.ppy.sh/users/3918650), [Ak1o](https://osu.ppy.sh/users/1600041), [Hivie](https://osu.ppy.sh/users/14102976), [Alchyr](https://osu.ppy.sh/users/4993032), [rubies87](https://osu.ppy.sh/users/4949934), [Briesmas](https://osu.ppy.sh/users/2865172), [UnagiDon](https://osu.ppy.sh/users/2062410), [Gamelan4](https://osu.ppy.sh/users/9856910), [Heaxys](https://osu.ppy.sh/users/5671417), [Ideal](https://osu.ppy.sh/users/3869519), [HiroK](https://osu.ppy.sh/users/4050738), [5\_5](https://osu.ppy.sh/users/6853438), [BabySnakes](https://osu.ppy.sh/users/4669728), [MTNTWarz](https://osu.ppy.sh/users/14036825), [Rinze](https://osu.ppy.sh/users/19901680), [cdh](https://osu.ppy.sh/users/2403621), [Cynplytholowazy](https://osu.ppy.sh/users/3901754) and [Horiiizon](https://osu.ppy.sh/users/8071438)), *\<\<\/nttld.:beings\>\> \~Truth in Uncertainty\~* is a visually stunning mapping masterpiece, helped along the way into Ranked by Beatmap Nominators [Axer](https://osu.ppy.sh/users/7299864) and [op45667](https://osu.ppy.sh/users/21973491).
```